config/default-imu_vn_100-params.yaml

```diff
@@ -52,3 +52,5 @@ imu_vn_100:
           mode: 1
           output: 3
           converge_rate: 5
+        ref:
+          use_models: false
```

Comment:
Nit: add "recalc_threshold" here as well (just to document it).

Reply:
Done in 321aff1f5c3eff46c4dc3a98e1dbab586690963e.